m4_sessions/1706-inning/group_one.md

```diff
@@ -1,9 +1,19 @@
 # Group One
 
-### Session Title:
+### Session Title: Looking Good: the importance of nonverbal communication
 
 ### Session Outcomes: 
+- Awareness of your own body language and how it affects others
+- Awareness of others' body language and how you respond
+- Trivia - tips and tricks
+- What some of the success you might have, losses you might suffer with body language
```

Comment:
Maybe even phrasing this around positive implications around understanding body language.  Something like...body language and how it can impact on your career.